### PR TITLE
Add `--progress` option to `provider_pull_db`

### PR DESCRIPTION
**Pull request by rmott-littler**
Adding a progress bar for `fin pull db`. `pv` already available in cli.

### CODE REVIEW

File: bin/fin [hunk 1]
```diff
@@ -2363,6 +2363,7 @@ show_help_pull ()
 	printh "--db-name=<dbname>" "Specify the DB Name to import into. (Defaults to default)"
 	printh "--force" "Generate and pull a new db dump, rather than using locally cached in /tmp"
 	printh "--remote-db=<remotedb>" "Specify the remote DB name to pull. (Used with Acquia)"
+	printh "--progress" "Show import progress."
```
Comment (sean-e-dietrich) [inline bin/fin:2366]: Remove `.` from the end

File: docs/content/fin/fin-help.md [hunk 2]
```diff
@@ -161,7 +161,8 @@ aliases:
 	  --db-name=<dbname>       	Specify the DB Name to import into. (Defaults to default)
 	  --force                  	Generate and pull a new db dump, rather than using locally cached in /tmp
 	  --remote-db=<remotedb>   	Specify the remote DB name to pull. (Used with Acquia)
-	
+	  --progress               	Show import progress.
```
Comment (sean-e-dietrich) [inline docs/content/fin/fin-help.md:164]: Please remove period from end of help text

File: docs/content/fin/fin-pull.md [hunk 3]
```diff
@@ -176,6 +176,7 @@ importing the data.
   --db-name=<dbname>             Specify the DB Name to import into. (Defaults to default)
   --force                       Force a new database file to be pulled
   --remote-db=<remote_db>       Specify the remote DB name to pull. (Used with Acquia)
+  --progress                    Show import progress.
```
Comment (sean-e-dietrich) [inline docs/content/fin/fin-pull.md:179]: Please remove period from end of help text

**Review by sean-e-dietrich — CHANGES_REQUESTED** [review]
Overall found this pretty useful and helpful! Just looking for those 3 changes to the help text.

**Comment by lmakarov** [comment]
ping @sean-e-dietrich

**Comment by lmakarov** [comment]
`pull` command was moved to the docksal/addons repo. See https://github.com/docksal/docksal/pull/1641 and https://github.com/docksal/addons/pull/78

**Pull request closed without merging**